src/main/java/org/opentestsystem/ap/imrt/common/model/Attachment.java

```diff
@@ -39,11 +39,16 @@
      */
     private Attachment() {}
 
-    public Attachment(final BaseItem item, final String fileType, final String fileName, final Instant uploadedDate) {
+    public Attachment(final BaseItem item,
+                      final String fileType,
+                      final String fileName,
+                      final Instant uploadedDate,
+                      final String updatedBy) {
```

Comment:
My lack of understanding some Spring magic is showing again here.  I found that if I didn't set the `updatedBy` field "manually", then the value wouldn't be set.  I suppose this constructor could actually be the culprit - maybe I should ditch this constructor and just use the getters/setters to get/set each individual field.